cmd/certificates/auth.go

```diff
@@ -257,7 +257,12 @@ func writeAuthExport(ctx context.Context, authDB *certificates.AuthorizationDB,
 
 	var authErrs errs.Group
 	for _, auth := range auths {
-		if err := w.Write([]string{email, auth.Token.String()}); err != nil {
+		isClaimed := "false"
+		if auth.Claim != nil {
```

Comment:
I'm not sure I follow the question. This helper function is used in the `certificates auth export` subcommand which generates a CSV of the specified authorizations/tokens (all, or by email address/list).

In this helper, `auths` is a slice of `Authorizations` which have a `*Claim` field. If the auth that comes out of the database has a nil claim then it has not been claimed, otherwise, it has and the claim has the details.

Does that answer your question?